pkgs/applications/science/logic/nusmv/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv
+, lib
+, fetchurl
+, autoPatchelfHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "NuSMV";
+  version = "2.6.0";
+
+  src = fetchurl {
+    url = "https://nusmv.fbk.eu/distrib/NuSMV-${version}-linux64.tar.gz";
+    sha256 = "1370x2vwjndv9ham5q399nn84hvhm1gj1k7pq576qmh4pi12xc8i";
+  };
+
+
+  nativeBuildInputs = [ autoPatchelfHook ];
+
+  installPhase = ''
```

Almost forgot:

Custom phases should be decorated:

```suggestion
  installPhase = ''
    runHook preInstall

```